// AUTOGENERATED FILE - DO NOT MODIFY!
// This file was generated by PyDjinni from 'error.pydjinni'
package test.error;

public abstract class ThrowingCallback {
    public abstract void throwingError() throws FooError;
}
